web/templates/twig-source/incuna-surveys/form/base/proportion-field.html

```diff
@@ -20,11 +12,15 @@
                         >
                     {% block input %}
                         <div class="proportion-input" integer-field model="model[$index]" id="field.id" form="form"></div>
+                        <div
+                            aif-slider-input
+                            model="model[$index]"
+                            ceiling="100"
```

Comment:
So currently the allocated amount (that's what's needed to set the with of the div) is available in the proportion field template with: `total - model[$index]`.